app/javascript/src/components/Dashboard/Contacts/Pane/Create.jsx

```diff
@@ -4,9 +4,9 @@ import { Pane, Typography } from "neetoui";
 
 import Form from "./Form";
 
-import { CONTACT_FORM_INITIAL_FORM_VALUES } from "../constants";
+import { INITIAL_FORM_VALUES } from "../constants";
 
-const NewContactPane = ({ showPane, setShowPane }) => {
+const NewPane = ({ showPane, setShowPane }) => {
```

Comment:
Go through https://github.com/bigbinary/bigbinary-website/issues/2462 and make necessary changes.

Reply:
Done.